### ISSUE TITLE
Feature request: sortier.onSave by language

### ISSUE THREAD

**Issue by jpaas** [snowcoders/sortier-vscode#76]
It would be nice to be able to configure which languages apply sortier.onSave e.g.
```
{
  "[javascript]": {
    "editor.formatOnSave": true,
    "sortier.onSave": true
  },
  "[javascriptreact]": {
    "editor.formatOnSave": true,
    "sortier.onSave": true
  }
}
```

**Comment by k2snowman69** [comment]
I think I'm going to reference the same answer I gave on https://github.com/snowcoders/sortier-vscode/issues/77 where if there was an exclude/include property I think it could handle both of these scenarios right?

**Comment by jpaas** [comment]
@k2snowman69 that would also work

**Comment by k2snowman69** [comment]
Sortier 2.6.1 added a .sortierignore file to disable sorting on specific files. This library will be updated to 2.6.1 with 
https://github.com/snowcoders/sortier-vscode/pull/126

**Comment by k2snowman69** [comment]
Released 1.2.3 so you will either need the following combinations:
 - Just sortier-vscode@1.2.3 + no sortier installed in the current project or parent project
 - Just sortier-vscode@1.2.* + sortier@2.6.1
